examples/data-objects/focus-tracker/src/FocusTracker.ts

```diff
@@ -12,17 +12,18 @@ import {
     IServiceAudience,
 } from "fluid-framework";
 
-export interface IFocusTrackerEvents extends IEvent {
-    (event: "focusChanged", listener: () => void): void;
+export interface IMouseFocusTrackerEvents extends IEvent {
+    (event: "focusChanged" | "mousePositionChanged", listener: () => void): void;
 }
 
 /**
  * Example of using the audience with signals to track focus state of connected clients
  * without writing changes to a DDS.
  */
-export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
+export class MouseFocusTracker extends TypedEventEmitter<IMouseFocusTrackerEvents> {
```

Comment:
I see that your approach here combines the functionality into a single class - did you consider having two separate classes (and what the pros/cons of that might be)?  I.e. a `FocusTracker` and a `MousePositionTracker`.

Reply:
Yeah, I think it definitely makes sense to split them up since the objects track completely separate events and have separate signal types.